packages/debugger/src/handler.ts

```diff
@@ -26,53 +26,50 @@ import { IDebugger } from './tokens';
 function updateIconButton(
   widget: DebuggerHandler.SessionWidget[DebuggerHandler.SessionType],
   onClick: () => void,
-  translator?: ITranslator
+  enabled?: boolean,
+  pressed?: boolean,
+  translator: ITranslator = nullTranslator
 ): ToolbarButton {
-  translator = translator || nullTranslator;
   const trans = translator.load('jupyterlab');
   const icon = new ToolbarButton({
     className: 'jp-DebuggerBugButton',
     icon: bugIcon,
-    tooltip: trans.__('Enable / Disable Debugger'),
+    tooltip: trans.__('Enable Debugger'),
+    ariaLabel: trans.__('Enable/Disable Debugger'),
```

Comment:
I am not sure if `aria-label` is needed in addition to `title` (which gets populated from `tooltip`). CC @manfromjupyter?

Reply:
From link shared by on accessibility call, there is a [rule](https://www.w3.org/TR/wai-aria-practices-1.1/#naming_rule_avoid_fallback) which seems to apply here:

> When authors do not specify an accessible name using an element or attribute that is intended for naming, browsers attempt to help assistive technology users by resorting to fallback methods for generating a name. For example, the HTML title and placeholder attributes are used as last resort sources of content for accessible names. Because the purpose of these attributes is not naming, their content typically yields low quality accessible names that are not effective.

So in that case I think that we should synchronise the `tooltip` (`title`) and `ariaLabel` (`aria-label`), i.e. both to show either `Enable Debugger` or `Disable Debugger`. Is this correct?

Reply:
I am using a screen reader (VoiceOver) in Binder to try this out, but I'm not sure I have any helpful feedback. Please note I'm not a regular or experienced screen reader user.

I have not been able to get focus on any of the sidebar buttons, and so I have not verified whether or not this label giving the desired behavior. I can set focus to the contents of the right sidebar and tab through it and it will tell me I am in the debugger section. Other buttons (like simple mode's toggle, all the buttons in the main toolbar, and the top of the file browser) are being read as their tooltips with the suffix of how to interact with them. This makes me think it is possible that the title is enough even though I can't test the specific buttons.

Reply:
@3coins I will drop the `aria-label` as the title has the same content and is changing depending on the button state (on the opposite of the `aria-label`)